shared/login/forms/intro.js

```diff
@@ -20,13 +20,19 @@ Intro.propTypes = {
 }
 
 export default connect(
-  state => ({justRevokedSelf: state.login.justRevokedSelf, loaded: state.login.loaded}),
+  state => ({
+    justLoginFromRevokedDevice: state.login.justLoginFromRevokedDevice,
+    justRevokedSelf: state.login.justRevokedSelf,
+    loaded: state.login.loaded
+  }),
   dispatch => ({
     onSignup: () => {
+      dispatch(setLoginFromRevokedDevice(''))
```

Comment:
Makes sense, filed a cleanup ticket for this: https://keybase.atlassian.net/browse/DESKTOP-1560